.gitignore

```diff
@@ -19,6 +19,3 @@ coverage
 
 # Webpack bundle analysis
 bundle-analysis
-
-# Babel generated files (from relay-compiler)
-__generated__
```

Comment:
I do worry that we're going to make PR diffs spew a bunch of useless stuff for each change we make. I think I remember this being best practice (committing the stuff into the repo) - what happens if we just re-create them on first compile?

Reply:
In practice I’ve not found it to be too much of an issue in previous projects (it’s rare for them to churn), though there will obviously be somewhat more noise, but mostly just around actual add/edit of queries/mutations — which is kinda useful?

The disadvantage of not committing them along is that we’re going to need to generate them again before compiles / flow checking can happen because we consume the generated types in the code;

https://github.com/buildkite/frontend/blob/2c8481aeb223101fa2fcc18c388a62d1b6226f97/app/components/team/Labels.js#L8-L12

Reply:
Seems legit